Source/WebCore/platform/mediastream/RealtimeMediaSource.cpp

```diff
@@ -1460,6 +1461,11 @@ void RealtimeMediaSource::getPhotoCapabilities(PhotoCapabilitiesHandler&& comple
     completion(PhotoCapabilitiesOrError("Not supported"_s));
 }
 
+auto RealtimeMediaSource::getPhotoSettings() -> Ref<PhotoSettingsPromise>
+{
+    return PhotoSettingsPromise::createAndReject("Not supported"_s);
```

Comment:
We should return a NotSupportedError here ideally.

Reply:
The spec says to return `OperationError` "If the data cannot be gathered for any reason", which we do